public/nginx.conf

```diff
@@ -53,7 +53,7 @@ http {
       }
 
       location /logger/ {
-        proxy_pass http://pz-logger.cf.piazzageo.io/;
+        proxy_pass http://pz-logger.stage.geointservices.io/;
```

Comment:
I haven't been having a problem with this in the new environment, so we might be able to take these hard coded urls out of this file entirely